### ISSUE TITLE
transition uri.neuinfo.org to run on nginx and point to interlex

### ISSUE THREAD

**Issue by tgbugs** [SciCrunch/NIF-Ontology#123]
The uri.neuinfo.org resolver is currently implemented on the old apache server using a rewrite rule to neurolex. This needs to be moved to nginx and needs to point to the interlex records. This will be implemented in a similar way to the ontology.neuinfo.org was [ported](https://github.com/tgbugs/pyontutils/tree/master/resolver) but without the need for the `#` replacement.

An additional complication is that all redirection of old identifiers that were moved to the scicrunch registry is handled by neurolex. We will need to find all of these identifiers (most should be in [scicrunch-registry.ttl](https://github.com/SciCrunch/NIF-Ontology/blob/master/scicrunch-registry.ttl)) and will need to redirect to the registry instead of to interlex.

The records of the mappings will be stored in `ttl/generated/`. The relationship for `http://uri.neuinfo.org -> http://scicrunch.org/resolver/` should be `owl:sameAs`. The relationship for `http://uri.neuinfo.org/nif/nifstd/ -> http://uri.interlex.org/origin/` should be `owl:sameAs`. In order for the semantics to actually be `owl:sameAs` we have to use use origin namespace that returns the view of the terms from the latest load of their source ontology (which means that it needs to be set up). Note that in order for the semantics to really be correct for this we need to finish merging the neurolex content back into the ontology. For cases where we decide not to merge back into the ontology I will point to the default namespace (see soon to be available specs doc for namespace definitions).

**Comment by jgrethe** [comment]
For the content negotiation:
html -> InterLex
RDF / OWL -> InterLex of GitHub?

Will need an RDF output for Registry items as well down the road then...

**Comment by tgbugs** [comment]
I think the rdf/owl should go to interlex since Troy has been writing code to return our proper owl translation for single terms and github cannot do that.

With respect to RDF output for the registry, I can adapt [scicrunch-registry.ttl](https://github.com/SciCrunch/NIF-Ontology/blob/master/scicrunch-registry.ttl) so that it can output individual records instead of translating the whole file so that all our various representations are consistent.

**Comment by jgrethe** [comment]
For the registry - we may want to have the portal return the RDF (build it from the inherent data)

**Comment by tgbugs** [comment]
That was my expectation. In theory we should be able to wrap my code so that it can be used transparently behind the scenes.

**Comment by jgrethe** [comment]
@tgbugs - so it wouldn't be pulling from scicrunch-registry.ttl then...

**Comment by tgbugs** [comment]
Yes. Had the wrong link. We would wrap the [code that generates](https://github.com/tgbugs/pyontutils/blob/master/pyontutils/scr_sync.py) scicrunch-registry.ttl from the database not load it again from the file.

**Comment by tgbugs** [comment]
The bulk of this work is done, there are ~5000 or so uris that still need to be reviewed and mapped to interlex.

**Comment by jgrethe** [comment]
Are we good to have Sean re-direct uri.neuinfo?  If so, we can then take down nif-apps1 and nif-apps2 I believe.

**Comment by tgbugs** [comment]
Yes.

**Comment by Aref-cs** [comment]
Hello everyone!

All turtle files I tried to access from [TTL directory](https://github.com/SciCrunch/NIF-Ontology/tree/master/ttl) have outdated NIFSTD URIs pointing to `http://uri.neuinfo.org/nif/nifstd`. 

Are there any updated files anywhere?

Regards,
Aref

**Comment by tgbugs** [comment]
The uri.neuinfo.org URIs are not outdated, they resolve directly to the master branch on github. See https://github.com/tgbugs/pyontutils/blob/6fa60b57fd225d211aa9aaf620aff547c72c09ef/nifstd/resolver/nif-ont-resolver.conf#L59 for reference.

**Comment by Aref-cs** [comment]
> The uri.neuinfo.org URIs are not outdated, they resolve directly to the master branch on github. See https://github.com/tgbugs/pyontutils/blob/6fa60b57fd225d211aa9aaf620aff547c72c09ef/nifstd/resolver/nif-ont-resolver.conf#L59 for reference.

Thanks a lot!
